### DISCUSSION TITLE
persistent notes per callsign

### DISCUSSION THREAD

**Discussion by DL2CC** [magicbug/Cloudlog#2943] [Ideas / Feature Requests]
When you repeatedly have a QSO with someone it would be good to quickly get reminders about information you have about this particular callsign.

In other words: make notes available per QSO partner callsign.

This way you can rememeber his name / nickname, spouse name, equipment he is using etc..
The notes box should be shown on the right side and quickly open after entering a callsign and should have an edit feature to quickly change the notes.

I've used this is DXLAB/DXKEEPER and it is a fantastic feature.

**Comment by nicolassolop** [comment]
I would love to have this feature!
